src/vsr/replica.zig

```diff
@@ -3897,7 +3901,7 @@ pub fn ReplicaType(
             assert(self.commit_min == self.commit_prepare.?.header.op);
             assert(self.commit_min <= self.commit_max);
 
-            if (self.status == .normal and self.primary()) {
+            if (self.status == .normal and self.primary() and !self.view_durable_updating()) {
```

Comment:
Hm, this feels wrong: if we bail out of this with `self.view_durable_updating()`, then at least we should chain-call this back once view_durable_update finishes, otherwise we won't re-start committing the pipeline. 

But i am wondreifg if we could just `assert` that we are not updating the view here? In this code ,we are primary, in normal, and we are already committing, so the view update must have happened already, no?

Reply:
Mhmmmm great catch _"we won't re-start committing the pipeline"_, I think I indeed got overly cautious here. I was worrying about a case wherein a primary reaches here without making its view durable. But that is not possible at all, since a primary starts off with nothing to commit (commit_min == commit_max), and now doesn't accept any new prepares till the view is durable. So if we've reached this point, we _should_ be guaranteed to have made our view durable!

Reply:
https://github.com/tigerbeetle/tigerbeetle/pull/2253/commits/3af24ebbffa0c5e6617865a83ed8c08ab9681dee